cpp/tensorrt_llm/kernels/decoderMaskedMultiheadAttention/decoderXQAImplJIT/nvrtcWrapper/CMakeLists.txt

```
@@ -17,7 +17,7 @@ add_dependencies(nvrtc_wrapper_src xqa_sources_h)
 target_include_directories(nvrtc_wrapper_src
                            PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/xqa_build)
 target_link_libraries(
-  nvrtc_wrapper_src PUBLIC ${NVRTC_LIB} ${NVRTC_BUILTINS_LIB} ${CUDA_DRV_LIB}
+  nvrtc_wrapper_src PUBLIC ${NVPTX_LIB} ${NVRTC_LIB} ${NVRTC_BUILTINS_LIB} ${CUDA_DRV_LIB}
                            ${CUDA_RT_LIB})
```

Please fix the formatting:
```suggestion
  nvrtc_wrapper_src PUBLIC ${NVPTX_LIB} ${NVRTC_LIB} ${NVRTC_BUILTINS_LIB}
                           ${CUDA_DRV_LIB} ${CUDA_RT_LIB})
```